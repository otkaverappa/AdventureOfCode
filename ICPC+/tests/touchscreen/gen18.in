19
rviirkzhzuhkclywbp 10
lftvuqdfrtcopwlyge
khojbvtdgscjhpodll
vqqolucbqqbxuleiwi
dqnlkpvtflwswtkohw
jmzzccyypfjlnmbaan
sxhzkfrjzdxgbgsaiu
egvtnefasidswvrfwc
kpljsksurkwzhbhcuv
ibxamcvizmpvqzkdlf
odzhpxiwypavmiwmli
ogsquhlljyowdezelq 10
btnalpvzatllectysn
idytdppgtqkfgmavol
lllngyykarlsgtyhmd
wbkrrwyxkztymglzts
yteampvsktbwyxzirq
hpqrqlsdtfeozchefv
vapgvrctqeehulxmep
zyusezgddpjikficld
tpzjtdtqoseuhetcyz
dhciwmrjtblghgxgrr
jkjzcptmvmqultdqbz 10
eukxyxffdclvvwhgvm
wqyrfqnqjqimsojelh
csohvbcqxmxuyvkyop
oeizwsnohwtufxntei
ujatvznvwatlrhpbgl
uvcbuxxzukvyupjxle
yazvaugsbxvjkpfmsb
jqafcvgxnpwawuaxpb
sxvtuqffimtanfspkv
nsuakqagkadcdvzyrw
rydzkwbzbuoopbgjdq 10
bfyngcplzrmsndqqea
ofbqbqgqrobxhecfsk
jhwlyidnnudtwuaymc
qsuhgwgpajvutgeprc
zxsorxhprkpdmfvgpe
evvhesbyzfpskppcfi
zpatzpwlvutmayhvho
qimpqdjaubeajfrlyq
bxdytwkvwtsdhiovze
bjgvmmxxrojsekrjkk
iwhepzkziawhezsmve 10
yucrjllpxezjqhfxnw
xzxfzwofvhrsnrnrlw
eylddmmtvrrjqoipvk
ljpjsjbiaqznmemahp
oukknbvfrdvmpiyfrs
otcqlcgaisapjqlvab
xviqbdfqmfxdxlyaek
ekkmemdperkfuhccaf
gfyukvxjjyjpkpzwdf
khuoaevuoywofctdwd
bwmmuybgpactfmczcc 10
fywtyshdxcjthmrwyn
wztlbxeijijokomgkk
ztpwwarfojbmwxnslq
prybajpkybtknsdeqb
hjhvukktixlvocmodm
yuzyxskkmnrerynyti
kddsdontscjwqjqpjq
jtavjtzarpanzkscfx
suqkwagnjzevpoorjx
llxeaztzkmdpjvjcfi
eoxppbkepzxbwkmwpo 10
vkphwswiniksqqipfx
qqbgpbjnlvlclhnaql
snvfxhzpxiedfutjck
klxxijctsrtienxauw
julherklofuqpgeogo
ziktbfeftdhnarjnyn
fkztrvlierwkiyusrx
xvcqbmgbdpodfvpgqg
deohxmtfknzemxbrpc
dxfinunurcckkfoanl
ogtbwsfiribhleeqot 10
mbpgfrqrxermrismjq
fqbwycfljmexhqzxyg
qoynurzobuakmhapeb
ulmdxqahjbghkwykms
bngdjjpwqsnwthifkh
ynohouoatokfhmupre
yicpapltxvbjdzwtgn
qwnjlyqumkjfqknsbq
iojhkkqplpircaorlb
pbvdoeieqyxrohgzoq
mghxvrpzsfrdgghdkx 10
judasarijzizrugyrd
qgdinunvcwymtkjxkb
zdkldukuosvfwlobvb
vkxzhxmchxztabxkmc
eyyurtapgosbrpoqox
oczxzbracomqstoqnh
knysdqwvfkluhzwgzy
jqylgkdyduorbagctk
spfabswiuuqtsajqnp
dtpinfzoiisbslsznt
rjdndvgwxrolitexct 10
fdjnlcodniecdylilq
duobldouyutanyewlp
ybvnlzroxcwlvcfjfs
ovmnripeitculdxgsj
hkzhoyulczvhrlegyv
oobyjdsuiscddmnctd
cnpeomnhxrnymdoncx
twuboyerktwfzyuqfk
cssbkfzyjpmnphjjia
jptwipbjqyzxibpadb
gecruogjvpufpfulbf 10
afosdqrlrhowkudmnz
ctkzkghcldnpipuzkx
rbkkkyjvumjjnoexpp
fwtsaghkxblhycllnv
jwsekepxsuwhjdeevg
mdqjeetfgeqvbzrufd
avcsrybadhgznscgbg
kwlqcdmeffakjahnuy
mvardgqsysycbiaobd
snjxujjflttfthcuyf
cpydjwhkhhzimtxvsr 10
gdxtwsyqzbmxiqogty
fciomjwyetwymcclya
dwsezecixsqsqxvblh
mjhtedrriteguieoog
vqqsjikbghctqrfykj
dbcnxhvtrchficyyuj
ghkoopkhgphqymtbaq
kxjbzrihtghqpqxcen
rquxfdpgqihqztpkup
beyxlincymeezxvwwa
zlgpwphvixheojjmiw 10
uvaujhzkeugbwiofzk
ujhfgpjuauiisffvzo
cyyiuhjrrawqmszuxh
ljdnfovytavsqxtofn
vreorcfdvgzsqkdtxj
hujawevmbodibazips
kuwfcxaskdojovexya
bvpejsnmsouhhhdfog
coymumwikcikelftqo
lfcevxleepjuxolxcf
kapwcxghjnabdoifsd 10
efikwtfthqrkxdmmbp
mjyxwaaboiginloxxm
rciaubmrezghouqoro
otpdbxlqkboiqhmyhh
zuyfvgmlaczrsnkhqn
geerhuzxblvkuugtzb
zlmapnthbgqtvxxbof
vpcycajxupswtrjhtz
vpiwvbrqypuowpfaqh
azgxqytlqctldoaomx
pdqpukfqbktrttsasj 10
ylwqqpctfehreyxunr
gskhedbyytyrewebpu
qrqwxzpbamxnefiqpm
tqmtjkkqjrranjtdhq
fxuflturzchorcedwp
qifbzwdniwspmzoiea
cbrbdbrwdvczmuisxj
qayywtqksetzexayyf
zpccmgeyaoqzxjzyjx
rzhleckizmixshowmc
cqbeetddcebobunigr 10
nqcmdkmxuctgfxwgdc
bgfgmhundhxjbkcfzh
plelnzrsypabrejxkv
geijnhtquxvtelekwt
jqohfqkxutueockyla
hgqbdmujzatxwenknv
cxsxqodgqqgdqokipp
umzvmstixiumdxkxwc
lzldprhhftqximjhix
cdiblcnrbxoxcaznfq
emymfqjqesxorzrzaf 10
cowfnkfrmegsulguzm
miercbftdztfextcej
ojcbnjvkwbevprftih
xpcaqvfwubyzmnioqw
zlgvomqedvxneufhwv
ecszfsyrhihxghkocz
bvdgscwxzbgvykzqje
jjxqsfqyocnsbqngwi
lufkvnhuxhmjnxslqm
sgngicamsouqwfldri
qzepiryvqtigfaouiz 10
wjlqzfhxnuaeesdjhn
cgjszrbgupbeozpbqo
jzlwvnczfiipvmvgfw
yieuxhaohqrzgaauyx
hbwplhegvbncznkdjk
njawzrxiuxcswjvuag
bepxhezgrmmawblwxn
puvjtxepibjjknpbmx
fnfwztyxujvrwlntwg
qcxbejkqyzrkywzesb
xsyudulahbvdhoghpm 10
szcqatazpbfjccdbyg
xkielejuspdhbvifli
ynjppoartdurkrdsxq
whnqwqxaohhbqipbxg
qazlfucpniklyisnzq
eysugcvykmzjsrjscp
ohgdpqopbidcbjavdi
xaihnkthbczgtppatg
sjvvsaxvjaqoiorrxe
dqlgvnmocbqxjlhfgb
