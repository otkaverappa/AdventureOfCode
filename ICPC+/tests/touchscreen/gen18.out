khojbvtdgscjhpodll 62
sxhzkfrjzdxgbgsaiu 66
vqqolucbqqbxuleiwi 66
kpljsksurkwzhbhcuv 69
egvtnefasidswvrfwc 71
dqnlkpvtflwswtkohw 72
lftvuqdfrtcopwlyge 78
ibxamcvizmpvqzkdlf 80
jmzzccyypfjlnmbaan 81
odzhpxiwypavmiwmli 90
yteampvsktbwyxzirq 53
lllngyykarlsgtyhmd 59
dhciwmrjtblghgxgrr 62
idytdppgtqkfgmavol 63
tpzjtdtqoseuhetcyz 74
hpqrqlsdtfeozchefv 77
btnalpvzatllectysn 78
zyusezgddpjikficld 78
wbkrrwyxkztymglzts 81
vapgvrctqeehulxmep 101
yazvaugsbxvjkpfmsb 67
jqafcvgxnpwawuaxpb 73
eukxyxffdclvvwhgvm 74
oeizwsnohwtufxntei 75
nsuakqagkadcdvzyrw 77
csohvbcqxmxuyvkyop 79
uvcbuxxzukvyupjxle 79
ujatvznvwatlrhpbgl 82
sxvtuqffimtanfspkv 83
wqyrfqnqjqimsojelh 92
bjgvmmxxrojsekrjkk 65
ofbqbqgqrobxhecfsk 65
qsuhgwgpajvutgeprc 68
jhwlyidnnudtwuaymc 72
qimpqdjaubeajfrlyq 72
bxdytwkvwtsdhiovze 73
evvhesbyzfpskppcfi 73
zxsorxhprkpdmfvgpe 76
bfyngcplzrmsndqqea 79
zpatzpwlvutmayhvho 85
otcqlcgaisapjqlvab 51
khuoaevuoywofctdwd 70
yucrjllpxezjqhfxnw 70
oukknbvfrdvmpiyfrs 74
xviqbdfqmfxdxlyaek 77
eylddmmtvrrjqoipvk 78
gfyukvxjjyjpkpzwdf 81
xzxfzwofvhrsnrnrlw 82
ekkmemdperkfuhccaf 89
ljpjsjbiaqznmemahp 93
fywtyshdxcjthmrwyn 61
jtavjtzarpanzkscfx 61
hjhvukktixlvocmodm 64
prybajpkybtknsdeqb 70
ztpwwarfojbmwxnslq 71
suqkwagnjzevpoorjx 73
kddsdontscjwqjqpjq 76
yuzyxskkmnrerynyti 77
llxeaztzkmdpjvjcfi 83
wztlbxeijijokomgkk 83
qqbgpbjnlvlclhnaql 63
fkztrvlierwkiyusrx 68
dxfinunurcckkfoanl 69
ziktbfeftdhnarjnyn 71
klxxijctsrtienxauw 75
deohxmtfknzemxbrpc 78
julherklofuqpgeogo 85
snvfxhzpxiedfutjck 94
xvcqbmgbdpodfvpgqg 98
vkphwswiniksqqipfx 106
pbvdoeieqyxrohgzoq 52
ulmdxqahjbghkwykms 54
fqbwycfljmexhqzxyg 58
qoynurzobuakmhapeb 66
mbpgfrqrxermrismjq 67
iojhkkqplpircaorlb 68
yicpapltxvbjdzwtgn 71
bngdjjpwqsnwthifkh 74
ynohouoatokfhmupre 75
qwnjlyqumkjfqknsbq 79
qgdinunvcwymtkjxkb 55
vkxzhxmchxztabxkmc 59
oczxzbracomqstoqnh 61
judasarijzizrugyrd 64
knysdqwvfkluhzwgzy 66
jqylgkdyduorbagctk 67
zdkldukuosvfwlobvb 70
eyyurtapgosbrpoqox 72
spfabswiuuqtsajqnp 77
dtpinfzoiisbslsznt 83
ybvnlzroxcwlvcfjfs 59
cnpeomnhxrnymdoncx 65
ovmnripeitculdxgsj 66
hkzhoyulczvhrlegyv 67
duobldouyutanyewlp 69
cssbkfzyjpmnphjjia 72
twuboyerktwfzyuqfk 72
jptwipbjqyzxibpadb 75
oobyjdsuiscddmnctd 75
fdjnlcodniecdylilq 77
fwtsaghkxblhycllnv 48
avcsrybadhgznscgbg 56
jwsekepxsuwhjdeevg 57
snjxujjflttfthcuyf 57
mdqjeetfgeqvbzrufd 62
afosdqrlrhowkudmnz 66
mvardgqsysycbiaobd 66
kwlqcdmeffakjahnuy 74
ctkzkghcldnpipuzkx 78
rbkkkyjvumjjnoexpp 80
beyxlincymeezxvwwa 60
vqqsjikbghctqrfykj 60
mjhtedrriteguieoog 65
fciomjwyetwymcclya 66
kxjbzrihtghqpqxcen 67
ghkoopkhgphqymtbaq 69
dwsezecixsqsqxvblh 72
dbcnxhvtrchficyyuj 80
gdxtwsyqzbmxiqogty 82
rquxfdpgqihqztpkup 83
cyyiuhjrrawqmszuxh 67
coymumwikcikelftqo 71
ljdnfovytavsqxtofn 72
kuwfcxaskdojovexya 73
vreorcfdvgzsqkdtxj 77
hujawevmbodibazips 81
bvpejsnmsouhhhdfog 82
ujhfgpjuauiisffvzo 84
uvaujhzkeugbwiofzk 92
lfcevxleepjuxolxcf 93
mjyxwaaboiginloxxm 56
otpdbxlqkboiqhmyhh 57
zuyfvgmlaczrsnkhqn 61
vpiwvbrqypuowpfaqh 66
geerhuzxblvkuugtzb 69
vpcycajxupswtrjhtz 69
azgxqytlqctldoaomx 72
efikwtfthqrkxdmmbp 73
rciaubmrezghouqoro 75
zlmapnthbgqtvxxbof 75
tqmtjkkqjrranjtdhq 62
cbrbdbrwdvczmuisxj 65
ylwqqpctfehreyxunr 65
gskhedbyytyrewebpu 66
fxuflturzchorcedwp 67
qayywtqksetzexayyf 78
rzhleckizmixshowmc 78
qrqwxzpbamxnefiqpm 85
zpccmgeyaoqzxjzyjx 87
qifbzwdniwspmzoiea 105
cxsxqodgqqgdqokipp 55
bgfgmhundhxjbkcfzh 58
geijnhtquxvtelekwt 60
nqcmdkmxuctgfxwgdc 62
hgqbdmujzatxwenknv 63
lzldprhhftqximjhix 68
cdiblcnrbxoxcaznfq 71
jqohfqkxutueockyla 71
umzvmstixiumdxkxwc 82
plelnzrsypabrejxkv 88
miercbftdztfextcej 57
zlgvomqedvxneufhwv 63
lufkvnhuxhmjnxslqm 66
bvdgscwxzbgvykzqje 73
ecszfsyrhihxghkocz 77
cowfnkfrmegsulguzm 78
jjxqsfqyocnsbqngwi 79
ojcbnjvkwbevprftih 79
sgngicamsouqwfldri 82
xpcaqvfwubyzmnioqw 85
hbwplhegvbncznkdjk 62
yieuxhaohqrzgaauyx 73
puvjtxepibjjknpbmx 74
wjlqzfhxnuaeesdjhn 76
fnfwztyxujvrwlntwg 77
qcxbejkqyzrkywzesb 77
jzlwvnczfiipvmvgfw 79
bepxhezgrmmawblwxn 81
cgjszrbgupbeozpbqo 85
njawzrxiuxcswjvuag 96
dqlgvnmocbqxjlhfgb 52
xaihnkthbczgtppatg 52
szcqatazpbfjccdbyg 60
xkielejuspdhbvifli 67
whnqwqxaohhbqipbxg 70
eysugcvykmzjsrjscp 73
qazlfucpniklyisnzq 74
sjvvsaxvjaqoiorrxe 77
ohgdpqopbidcbjavdi 78
ynjppoartdurkrdsxq 81
